packages/cdk8s-cli/templates/java-app/cdk8s.yaml

```diff
@@ -0,0 +1,4 @@
+language: java
+app:  mvn exec:java -Dexec.mainClass="com.mycompany.app.Main"
```

Comment:
This might be a bit tricky to find when users modify this package name. Curious what are we doing in the AWS CDK...